packages/config/src/layer2s/mode.ts

```diff
@@ -25,4 +35,52 @@ export const mode: Layer2 = upcoming({
       ],
     },
   },
+  upgradeability,
+  l1StandardBridgeEscrow: EthereumAddress(
+    '0x735aDBbE72226BD52e818E7181953f42E3b0FF21',
+  ),
+  rpcUrl: 'https://mainnet.mode.network/',
+  sequencerAddress: EthereumAddress(
+    discovery.getContractValue('SystemConfig', 'batcherHash'),
+  ),
+  inboxAddress: EthereumAddress('0x24e59d9d3bd73ccc28dc54062af7ef7bff58bd67'),
+  genesisTimestamp: new UnixTime(1700125343),
+
+  l2OutputOracle: discovery.getContract('L2OutputOracle'),
+  portal: discovery.getContract('OptimismPortal'),
+  //stateDerivation: DERIVATION.OPSTACK('MODE'),
+  isNodeAvailable: true,
+  milestones: [
+    {
+      name: 'Mode Network Mainnet Launch',
+      link: 'https://twitter.com/modenetwork/status/1752760726907760933',
+      date: '2024-01-31T00:00:00Z',
+      description: 'Mode Network is live on mainnet.',
+    },
+  ],
+  knowledgeNuggets: [],
+  roleOverrides: {
+    batcherHash: 'Sequencer',
+    PROPOSER: 'Proposer',
+    GUARDIAN: 'Guardian',
+    CHALLENGER: 'Challenger',
+  },
+  nonTemplatePermissions: [
+    ...discovery.getMultisigPermission(
+      'ModeMultisig',
+      'This address is the owner of the following contracts: ProxyAdmin, SystemConfig. It is also designated as a Guardian of the OptimismPortal, meaning it can halt withdrawals. It can upgrade the bridge implementation potentially gaining access to all funds, and change the sequencer, state root proposer or any other system component (unlimited upgrade power).',
+    ),
+    ...discovery.getMultisigPermission(
+      'ChallengerMultisig',
+      'This address is the permissioned challenger of the system. It can delete non finalized roots without going through the fault proof process.',
```

Comment:
```suggestion
      'This address is the permissioned challenger of the system. It can delete non finalized roots without going through the fault proof process. It is also designated as the Guardian.',
```